lxd/apparmor/feature_check.go

```diff
@@ -0,0 +1,78 @@
+package apparmor
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"text/template"
+
+	"github.com/google/uuid"
+
+	"github.com/canonical/lxd/lxd/sys"
+	"github.com/canonical/lxd/shared"
+)
+
+var featureCheckProfileTpl = template.Must(template.New("featureCheckProfile").Parse(`#include <tunables/global>
+profile "{{ .name }}" flags=(attach_disconnected,mediate_deleted) {
+  #include <abstractions/base>
+  #include <abstractions/consoles>
+  #include <abstractions/nameservice>
+
+{{- if eq .feature "mount_nosymfollow" }}
+  mount options=(nosymfollow) /,
+{{- end }}
+
+}
+`))
```

Comment:
```suggestion
var featureCheckProfileTpl = template.Must(template.New("featureCheckProfile").Parse(`#
profile "{{ .name }}" flags=(attach_disconnected,mediate_deleted) {

{{- if eq .feature "mount_nosymfollow" }}
  mount options=(nosymfollow) /,
{{- end }}

}
`))
```

It's a nitpick but I'd prefer going with an even simpler profile as those abstractions are quite large (they include other files):

```
$ apparmor_parser --preprocess aa.profile | wc -l
1021
$ apparmor_parser --preprocess aa-light.profile | wc -l
3
```

Reply:
Thanks, fixed!